src/orqviz/gradients.py

```diff
@@ -49,15 +50,15 @@ def _gradient_function(pars: ParameterVector, direction: np.ndarray) -> float:
     grad = np.zeros_like(params)
 
     if stochastic:
-        direction = np.random.choice([0.0, 1.0], size=np.shape(params))
+        direction = np.random.choice([-1.0, 1.0], size=np.shape(params))
         grad_value = gradient_function(params, direction)
-        grad += grad_value * direction
+        grad = grad + grad_value * direction
```

Comment:
Why this change?

Reply:
Manipulating in place has caused some silly problems in the past.